R/save_arcadia_plot.R

```diff
@@ -0,0 +1,44 @@
+#' Save Arcadia Plot
+#'
+#' @param plot ggplot object to be saved
+#' @param filepath The path where the plot will be saved
+#' @param orientation The orientation of the plot, either "landscape" or "portrait"
+#'
+#' @return None
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' p <- ggplot(mtcars, aes(x = hp, y = mpg)) + geom_point()
+#' save_arcadia_plot(p, "plot.pdf", "landscape")
+#' }
+save_arcadia_plot <- function(filepath, plot, orientation = c("landscape", "portrait")) {
+  # Ensure the orientation argument is valid
+  orientation <- match.arg(orientation)
+
+  # Set size based on orientation
```

Comment:
In the future, it would be worthwhile to have these match the standard panel sizes in the style guide.

Reply:
Should I name them `full_wide`, `float_wide` etc?

Reply:
I went ahead and changed this to `panel_size` and the options/sizes match those in the style guide in pixels. I was uncertain about the height measurements though because the style guide only seems to give widths

Reply:
Yeah, I think we should keep things consistent. We use those terms in the `arcadia-pycolor` implementation, e.g. [here](https://github.com/Arcadia-Science/arcadia-pycolor/blob/6f940d815d6195972bb4bdbe2544c3894098b128/arcadia_pycolor/style_defaults.py#L15C1-L30C1). 

```python
# Common figure sizes for Arcadia Creative Cloud library templates.
FULL_WIDE = (13.33333, 5.27777)
FULL_SQUARE = (6.52777, 6.52777)
FLOAT_WIDE = (9.16667, 5.27777)
FLOAT_SQUARE = (4.44444, 4.44444)
HALF_SQUARE = (6.38888, 6.38888)

# Dictionary of common figure sizes.
FIGURE_SIZES = {
    "full_wide": FULL_WIDE,
    "full_square": FULL_SQUARE,
    "float_wide": FLOAT_WIDE,
    "float_square": FLOAT_SQUARE,
    "half_square": HALF_SQUARE,
}
```

Reply:
Note that the numbers for sizes above are at 72dpi:

13.33333 * 72 ~= 960px, +2 * 20px padding around plots = 1000px wide for `FULL_WIDE`